src/components/item/sharing/VisibilitySelect.js

```diff
@@ -254,7 +245,7 @@ const VisibilitySelect = ({ item, edit }) => {
             {t('Pseudonymized')}
           </MenuItem>
           <MenuItem value={SETTINGS.ITEM_PUBLIC.name}>{t('Public')}</MenuItem>
-          <MenuItem value={SETTINGS.ITEM_PUBLISHED.name}>
+          <MenuItem value={SETTINGS.ITEM_PUBLISHED.name} disabled>
```

Comment:
Remove.

Reply:
This can cause a problem that when item is published, the selec dropdown menu will be empty